arxivdigest/api/README.md

```diff
@@ -219,21 +219,29 @@ Fields are returned in a JSON in the format, the topics are sorted descending by
     {
       "user_feedback": {
         user_id: {
-           date: [
-            {topic: feedback},
-            {topic: feedback},
-           ]
+          {topic: feedback},
```

Comment:
not valid json, probably should be something like
user_id: {
              topic: feedback_dict,
              topic: feedback_dict,
             }